src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -24,6 +24,12 @@
   </ItemGroup>
 
   <!-- Dependencies -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Internal.VisualStudio.Shell.Interop.14.0.DesignTime" Version="14.3.25407-alpha" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.15.0" Version="16.7.30329.88" />
+    <PackageReference Include="Microsoft.ServiceHub.Framework" Version="2.7.313-preview" />
```

Comment:
Please don't specify versions here. We use `Packages.targets` file for that.

Reply:
I've seen this on other projects (using a separate file for packages, like my previous team's [Packages.Data.props](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props)). Is there a way to work with the VS team to have this be a recognizable pattern? Because the tooling does this automatically, so a 'bad path' is the default for this pattern. 😔

Reply:
There has been some investigation into this, yes. Chat to @swesonga for more info.